tlatools/org.lamport.tlatools/customBuild.xml

```diff
@@ -467,219 +473,131 @@
-					<exclude name="tlc2/tool/distributed/**/*Test*.java" />
-					
-					<exclude name="**/PCalTest.java" />
-					<exclude name="**/SANYTest.java" />
-					<exclude name="**/*_TTraceTest.java" />
-					<exclude name="**/CommonTestCase.java" />
-					<exclude name="**/ModelCheckerTestCase.java" />
-					<exclude name="**/TTraceModelCheckerTestCase.java" />
-					<exclude name="**/TLCDebuggerTestCase.java" />
-					<exclude name="**/PCalModelCheckerTestCase.java" />
-					<exclude name="**/TraceExpressionSpecTest.java" />
-					<exclude name="**/TraceExpressionSpecSafetyTest.java" />
-					<exclude name="**/SuiteTestCase.java" />
-					<exclude name="**/SuiteETestCase.java" />
-					<exclude name="**/SuccessfulSimulationTestCase.java" />
-					<exclude name="**/AbstractExampleTestCase.java" />
-					<exclude name="**/AbstractCoverageTest.java" />
-					<exclude name="**/InitEvalOrderTest.java" />
-					<exclude name="**/TestMPRecorder.java" />
-					<exclude name="**/TestPrintStream.java" />
-					<exclude name="**/BidirectionalTransitions1BTest.java" />
-					<exclude name="**/BidirectionalTransitions2CTest.java" />
-					<exclude name="**/Abstract*Test.java" />
-					<exclude name="**/TestDriver.java" />
-					<exclude name="**/TestDriver2.java" />
-					<exclude name="**/AllTests.java" />
-					<exclude name="util/IsolatedTestCaseRunner.java"/>
-					<!-- Known test failures -->
-					<exclude name="**/pcal/StackTestTest.java" />
-					<exclude name="**/pcal/TestPCandStackTest.java" />
-					<!-- These take too long -->
-					<exclude name="**/MacroQuicksortTest.java" />
-					<exclude name="**/MacroRealQuicksortTest.java" />
-					<exclude name="**/PcalPaxosTest.java" />
-					<exclude name="**/DetlefsTest.java" />
-					<exclude name="**/StarkMutexTest.java" />
-					<exclude name="**/SimpleMultiProcTest.java" />
-				</fileset>
-			</batchtest>
-			<batchtest fork="yes" todir="${test.reports}">
-				<fileset dir="${test.dir}">
-					<!-- Run TTrace tests after the original tests are run -->	
-					<include name="**/*_TTraceTest.java" />
-				</fileset>
-			</batchtest>
-			<batchtest fork="yes" todir="${test.reports}">
-				<fileset dir="${test.dir}">
-					<include name="tlc2/tool/EvaluatingValueTest.java" />
-				</fileset>
-			</batchtest>
-			<batchtest fork="yes" todir="${test.reports}">
-				<fileset dir="${test.dir}">
-					<include name="tlc2/tool/UserModuleOverrideAnnotationTest.java" />
-				</fileset>
-			</batchtest>
-			<batchtest fork="yes" todir="${test.reports}">
-				<fileset dir="${test.dir}">
-					<include name="tlc2/tool/UserModuleOverrideFromJarTest.java" />
-				</fileset>
-			</batchtest>
-			<batchtest fork="yes" todir="${test.reports}">
-				<fileset dir="${test.dir}">
-					<include name="tlc2/tool/UserModuleOverrideTest.java" />
-				</fileset>
-			</batchtest>
-		</junit>
-
-		<!-- remove copied class.dir -->
+        <jacoco:coverage destfile="${code-coverage-seq.exec}"
```

Comment:
I suspect this large diff is due to the replacement of tabs with spaces; the customBuild.xml file uses tabs for indentation, could you check that?

Reply:
thanks I'll take a look and keep diff minimal! I think the diff is weird also due to the fact that I've removed some tests to make the ci complete faster while I'm still working on this